docs/source/release_notes.rst

```diff
@@ -8,6 +8,7 @@ Release Notes
         * Sped up permutation importance for some pipelines :pr:`1762`
     * Fixes
     * Changes
+        * Added ``random_seed`` as an argument to our automl/pipeline/component api. Using ``random_state`` will raise a warning :pr:`1798`
```

Comment:
We probably can't add this to `breaking_changes`, but is there another category similar that we can put it in to warn users? Just wondering since it'll soon be a breaking change once we officially deprecate.

Reply:
I think this is fine for now? Here we're telling users we'll raise a warning and the warning itself will tell them about the impending deprecation.